orca-kayenta/src/main/kotlin/com/netflix/spinnaker/orca/kayenta/tasks/MonitorKayentaCanaryTask.kt

```diff
@@ -52,7 +52,12 @@ class MonitorKayentaCanaryTask(
       val canaryScore = canaryResults.result!!.judgeResult.score.score
 
       return if (canaryScore <= context.scoreThresholds.marginal) {
-        TaskResult(TERMINAL, mapOf(
+        val resultStatus = if (stage.context["continuePipeline"] == true) {
```

Comment:
Can it just be `stage.context["continuePipeline"]`?

Reply:
Kotlin seems to prefer that the type of the expression in an if is strictly Boolean.  Using just `stage.context["continuePipeline"]` on its own is an `Any?` type